### ISSUE TITLE
Informix 11.7 Schema's hidden when not included in connection string's Database/Schema

### ISSUE THREAD

**Issue by nomadno** [dbeaver/dbeaver#8987]
#### System information:
- Operating system (distribution) and version:  Windows 10 Pro (1909)
- DBeaver version:  7.0.0.202002292132
- Java version:  Java SE Version 7 Update 55 (build 1.7.0_55-b14)
- Additional extensions

#### Connection specification:
- Database name and version:  Informix Dynamic Server 11.70.FC7
- Driver name:  IBM Informix JDBC Driver for IBM Informix Dynamic Server 3.70.JC2DE
- Do you use tunnels or proxies (SSH, SOCKS, etc)?:  Not through DBeaver - IPSec and L2TP

#### Steps to reproduce, if exist:
1 - Create a connection to an Informix database which hosts more than one database and share some but not all schema names.
2 - Open the database which is not in the database connection string and includes schema names not in the database used in the connection string and expand the database to view the available schemas.
Schema's not in the connection string's Database/Schema are not viewable in the selected database.

This was seen on a production database which had nearly identical schemas across all of the databases.

No errors were seen.

**Comment by michbudz** [comment]
I confirm this problem.
I have reported a similar problem [issue 8572](https://github.com/dbeaver/dbeaver/issues/8572)
Is there any chance to fix it?

**Comment by uslss** [comment]
sorry for the late response. Thanks for the bug report

**Comment by LonwoLonwo** [comment]
We have updated the driver version, now some things work better.
The procedure list now displays both procedures and functions.
But now Informix main support is in [Dbeaver-EE](https://dbeaver.com/download/) version.
In EE-version we made the process of reading diagrams a little different, and now for the system database and the main database specified in the connection - the schemes are read correctly. Other databases will give an error when trying to open schemas. 

Use separate connections for separate databases in this case. 

Perhaps this system will be improved in the future.
